harness/determined/searcher/search_runner.py

```diff
@@ -98,6 +114,13 @@ def run(
                         )
                         experiment_is_active = False
                         break
+                    elif e.trialProgress:
```

Comment:
looksl ike we need to have the trial_progress state updated when handling a TrialProgress event

Reply:
👍